perses/tests/test_repex.py

```diff
@@ -188,5 +185,218 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
+    from perses.app.relative_setup import RelativeFEPSetup
+    from perses.samplers.multistate import HybridRepexSampler
+    from perses.annihilation.relative import RESTCapableHybridTopologyFactory
+
+    from openmmtools.multistate import MultiStateReporter, MultiStateSamplerAnalyzer
+    from openmmtools import cache, utils, mcmc
+    platform = configure_platform(utils.get_fastest_platform().getName())
+
+    data = {}
+    n_iterations = 3000
+    n_states = 12
+    transformations = [(0, 1), (1, 0)]  # Ligand indices in perses/data/test.smi
+
+    with enter_temp_directory() as temp_dir:
+        for ligand_A_index, ligand_B_index in transformations:
+            # Generate topology proposal and positions
+            smiles_filename = resource_filename("perses", os.path.join("data", "test.smi"))
+            fe_setup = RelativeFEPSetup(
+                ligand_input=smiles_filename,
+                old_ligand_index=ligand_A_index,
+                new_ligand_index=ligand_B_index,
+                forcefield_files=["amber14/tip3p.xml"],
+                small_molecule_forcefield="gaff-2.11",
+                phases=["solvent"],
+                solvent_padding=1.7 * unit.nanometers)
+
+            # Generate htf
+            htf = RESTCapableHybridTopologyFactory(
+                topology_proposal=fe_setup.solvent_topology_proposal,
+                current_positions=fe_setup.solvent_old_positions,
+                new_positions=fe_setup.solvent_new_positions
+            )
+
+            # Make sure LRC is set correctly
+            hybrid_system = htf.hybrid_system
+            force_dict = {force.getName(): index for index, force in enumerate(hybrid_system.getForces())}
+            custom_force = hybrid_system.getForce(force_dict['CustomNonbondedForce_sterics'])
+            nonbonded_force = hybrid_system.getForce(force_dict['NonbondedForce_sterics'])
+            custom_force.setUseLongRangeCorrection(False)
+            nonbonded_force.setUseDispersionCorrection(True)
+
+            # Create a reporter
+            selection = "not water"
+            reporter_file = os.path.join(temp_dir, f"{ligand_A_index}_{ligand_B_index}.nc")
+            reporter = MultiStateReporter(
+                reporter_file,
+                analysis_particle_indices=htf.hybrid_topology.select(selection),
+                checkpoint_interval=10)
+
+            # Build the hybrid repex sampler
+            sampler = HybridRepexSampler(
+                mcmc_moves=mcmc.LangevinDynamicsMove(
+                    timestep=4.0 * unit.femtoseconds,
+                    collision_rate=1.0 / unit.picosecond,
+                    n_steps=50,
+                    reassign_velocities=False,
+                    n_restart_attempts=20,
+                    constraint_tolerance=1e-06),
+                replica_mixing_scheme='swap-all',
+                hybrid_factory=htf,
+                online_analysis_interval=10)
```

Comment:
Should `online_analysis_interval` be on here? And should it be so frequent?

Reply:
We should actually set it to 0, since we don't need the results of online analysis for the test.

Reply:
Should it be `0` or `None`?